docs/source/contrib_docs/doc-new-repos.md

```diff
@@ -0,0 +1,43 @@
+# Setting up a README in a new Jupyter repo
+
+Providing users and developers consistency across repos is a valuable time
+saver and improves user productivity.
+
+On a larger scope, having the Jupyter name appear prominently in a repo's
+`README.md` file improves the project's name awareness.
+
+## Recommended elements in Jupyter project repos
+
+### Link in repo description
+Please include a link to the documentation in the repo's description.
+
+![Repo description and documentation link](static/repo-description.png "Screenshot of documentation link in GitHub repo description")
+
+### Badges in README
+One common way that individuals find documentation is to look for and click
+on the doc badge that commonly is found right after the title. Another
+benefit is an easy visual indication if the docs are not rendering properly.
+
+![Badges in README.md](static/repo-badges.png "Screenshot of badges displayed under the repo title in the README file")
+
+### Resources section in README
```

Comment:
I'm a bit concerned that these docs are quite prescriptive and detailed, and risk going out of date. The more info there is in docs, the less likely it is that we'll remember to change it.